tensorflow/training/docker/2.4/py3/Dockerfile.cpu.os_scan_allowlist.json

```diff
@@ -0,0 +1,328 @@
+{
+    "avahi": [
+        {
+            "name": "CVE-2021-3468",
+            "description": "A flaw was found in avahi in versions 0.6 up to 0.8. The event used to signal the termination of the client connection on the avahi Unix socket is not correctly handled in the client_work function, allowing a local attacker to trigger an infinite loop. The highest threat from this vulnerability is to the availability of the avahi service, which becomes unresponsive after this flaw is triggered.",
+            "uri": "http://people.ubuntu.com/~ubuntu-security/cve/CVE-2021-3468",
+            "severity": "MEDIUM",
+            "attributes": [
+                {
+                    "key": "package_version",
+                    "value": "0.7-3.1ubuntu1.2"
+                },
+                {
+                    "key": "package_name",
+                    "value": "avahi"
+                },
+                {
+                    "key": "CVSS2_VECTOR",
+                    "value": "AV:L/AC:L/Au:N/C:N/I:N/A:P"
+                },
+                {
+                    "key": "CVSS2_SCORE",
+                    "value": "2.1"
+                }
+            ]
+        }
+    ],
+    "gcc-7": [
+        {
+            "name": "CVE-2020-13844",
+            "description": "Arm Armv8-A core implementations utilizing speculative execution past unconditional changes in control flow may allow unauthorized disclosure of information to an attacker with local user access via a side-channel analysis, aka \"straight-line speculation.\"",
+            "uri": "http://people.ubuntu.com/~ubuntu-security/cve/CVE-2020-13844",
+            "severity": "MEDIUM",
+            "attributes": [
+                {
+                    "key": "package_version",
+                    "value": "7.5.0-3ubuntu1~18.04"
+                },
+                {
+                    "key": "package_name",
```

Comment:
package_name is already defined on line 28

Reply:
Isn't this lot of info in code, could the below format be okay?
```
[
                {
                    "package": "gcc-7",
                    "versions": ["7.5.0-3ubuntu1~18.04"],
                    "CVEs": ["CVE-2020-13844"],
                    "reason": "New version not yet available.",
                },
                {
                    "package": "tensorflow",
                    "versions": ["2.3.2"],
                    "CVEs": ["CVE-2020-13844"],
                    "reason": "New version not yet available.",
                }
]
```

Reply:
I want to make the task of whoever updates this file as simple as Copying and Pasting a Scan Result for an image. By changing the format to be more concise, we improve the formatting, but at the cost of more time taken to update these vulnerabilities, as well as losing the ability to directly (through code or even visually) compare the `allowlist` with the `scan_results` from the scanner.

Reply:
Ohok, I thought its format defined by you. Lets keep as it is then.